frontend/packages/dev-console/src/components/import/serverless/ServerlessRouteSection.tsx

```diff
@@ -72,16 +84,30 @@ const ServerlessRouteSection: React.FC = () => {
         options={portOptions}
       />
       {domainMappingLoaded || domainMappingLoadErr ? (
-        <SelectInputField
-          data-test-id="kafkasource-bootstrapservers-field"
-          name="serverless.domainMapping"
-          label={t('devconsole~Domain mapping')}
-          options={domainMappingResources}
-          placeholderText={t('devconsole~Add domain')}
-          helpText={t('devconsole~Mapping a custom domain to the Knative service')}
-          isCreatable
-          hasOnCreateOption
-        />
+        <>
+          <SelectInputField
+            data-test-id="domain-mapping-field"
+            name="serverless.domainMapping"
+            label={t('devconsole~Domain mapping')}
+            options={domainMappingResources}
+            placeholderText={t('devconsole~Add domain')}
+            helpText={t('devconsole~Enter custom domain to map to the Serverless Deployment')}
```

Comment:
```suggestion
            helpText={t('devconsole~Enter custom domain to map to the Serverless deployment')}
```

Reply:
Updated the text based on the discussion over the tmp-odc-serverless slack channel.